assets/javascript/app.js

```diff
@@ -0,0 +1,582 @@
+    var validInput = true;
+    // var longitude = -81.667;
```

Comment:
As discussed longitude and latitude were predefined here so that default results are for cleveland